.travis.yml

```diff
@@ -8,7 +8,7 @@ notifications:
     email: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.checkout("InterestRates"); Pkg.add("Coverage"); Pkg.test("InterestRates"; coverage=true)'
```

Comment:
Yep. The failures on the tag builds were due to one of those annoying git issues with being on a detached head. Travis does the work for you - plus for PR's, you want Travis to run on the PR version of the code, not master.